ush/config.sh.RRFS_CONUS_3km

```diff
@@ -182,6 +182,11 @@ COMINgfs=""
 STMP="YourOwnSpace/NCO_dirs/stmp"  # Path to directory STMP that mostly contains input files.
 PTMP="YourOwnSpace/NCO_dirs/ptmp"  # Path to directory STMP that mostly contains input files.
 NWGES="YourOwnSpace/NCO_dirs/nwges"  # Path to directory NWGES that save boundary, cold initial, restart files
+if [[ ${REGIONAL_ENSEMBLE_OPTION} == "5" ]]; then
+  RRFSE_NWGES="YourOwnSpace/NCO_dirs/nwges"  # Path to RRFSE directory NWGES that mostly contains ensemble restart files for GSI hybrid.
+  NUM_ENS_MEMBERS=10     # FV3LAM ensemble size for GSI hybrid analysis
+fi
+
 
 if [[ ${DO_ENSEMBLE}  == "TRUE" ]]; then
    NUM_ENS_MEMBERS=9
```

Comment:
Is line 192 conflict with line 187?
   NUM_ENS_MEMBERS=9
   NUM_ENS_MEMBERS=10

Reply:
They are not related and both random numbers here.